intro-to-rails-4/betsy.checkpoint.md

```diff
@@ -0,0 +1,58 @@
+# bEtsy
+
+<!-- >>>>>>>>>>>>>>>>>>>>>> BEGIN CHALLENGE >>>>>>>>>>>>>>>>>>>>>> -->
+<!-- Replace everything in square brackets [] and remove brackets  -->
+
+
+### !challenge
+
+* type: short-answer
+* id: 79e941d4-f91a-4aa9-8256-6a4b2195d669
+* title: bEtsy Pull Request
+* points: 3
+* topics: rails
+
+##### !question
+
+**ONE** team member please fork and clone the [Ada-C14 bEtsy Ranker Repo](https://github.com/Ada-C14/betsy).
+
+Then cd into the folder with `cd betsy`
+
+Then add your team as collaborators on the repository.
+
+Then create a new rails project with `rails new .`, add commit and push to master.
+
+Then complete the project and create a pull request. 
+
+When you create the pull request your tests will automatically run and you can see the results at the bottom of the PR.
+
+![Pull Request](images/automatic-tests.png)
+
+Notice that it says, **"All checks have passed".** You can  click on **Show all checks** and see the results of the tests.
+
+Please submit a link to your pull request here.
+
+**Due:  Monday November 16th 9am**
```

Comment:
Not a real review since I'm just glancing at this, but I think the due date is Wed November 25th.